src/main/webapp/src/app/app-routing.module.ts

```diff
@@ -30,6 +31,7 @@ const routes: Routes = [
   {path: 'page/index', component: IndexComponent},
   {path: 'page/test-page1', component: TestPage1Component},
   {path: 'page/test-page2', component: TestPage2Component},
+  {path: 'page/schedule-page', component: SchedulePageComponent},
```

Comment:
Can it be `schedule` instead of `schedule-page`?

Reply:
Sure thing! I'll make the change